package.json

```diff
@@ -21,48 +21,78 @@
     "test": "tests"
   },
   "scripts": {
-    "build": "ember build",
-    "start": "ember server",
-    "test": "ember test",
+    "build": "ember build --environment=production",
+    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel \"lint:!(fix)\"",
+    "lint:fix": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*:fix",
+    "lint:hbs": "ember-template-lint .",
+    "lint:hbs:fix": "ember-template-lint . --fix",
+    "lint:js": "eslint . --cache",
+    "lint:js:fix": "eslint . --fix",
+    "start": "ember serve",
+    "test": "npm-run-all lint test:*",
+    "test:ember": "ember test",
+    "test:ember-compatibility": "ember try:each",
     "test:all": "ember try:each"
   },
   "dependencies": {
     "ember-cli-babel": "^7.26.10",
     "ember-getowner-polyfill": "^2.0.1"
```

Comment:
This polyfill can go since the minimum ember is now 3.24 and not < 2.3

Reply:
Removed in https://github.com/ember-intl/cp-validations/pull/90 via https://github.com/ember-intl/cp-validations/pull/90/commits/df81f36b54eff5553ce8177c62bad940c7cad449